### DISCUSSION TITLE
Is there a way to export the tile matrix after making pseudobulk replicates?

### DISCUSSION THREAD

**Discussion by alyamahmoud** [GreenleafLab/ArchR#1214] [Questions / Documentation]
Hi Ryan

I am trying to export the tile matrix after making pseudobulk replicates. 

> proj.ODC <- addGroupCoverages(ArchRProj = proj, groupBy = "Sample.ID")
> proj.ODC <- addPeakMatrix(proj, force=TRUE)
> tile <- getMatrixFromProject(proj, 'TileMatrix', binarize = TRUE)

I am expecting a tile matrix with 'Sample.ID' as colnames but instead I am still getting the sample#cellBarcode (not psuedobulked)

> tile
class: SummarizedExperiment 
dim: 6062095 69794 
metadata(0):
assays(1): TileMatrix
rownames: NULL
rowData names(3): seqnames idx start
colnames(69794): SRR14514130#GGAACCCAGGTCACTT-1
  SRR14514130#TTTGAGGGTTTAAGGA-1 ... SRR14514155#CCTTGCATCACTAGGT-1
  SRR14514155#ACTGTCCCAATACTGC-1
colData names(25): BlacklistRatio DoubletEnrichment ... Sex
  Sample.ID

any tips on how to check for that? 
Thank you

**Comment by rcorces** [comment]
`getMatrixFromProject()` returns an actual matrix stored in the Arrow files.
You are looking for `getGroupSE()` but I think asking for the tile matrix might blow up memory.
https://www.archrproject.com/reference/getGroupSE.html

Starting today, I'm on paternity leave. This is the last post I will make and cannot reply further. I also wont be replying to new posts until late January.
https://github.com/GreenleafLab/ArchR/discussions/1215
